### ISSUE TITLE
我想在conv1之后插入一个BN层，虽然保存好模型但是不能用，出现一些错误，请问怎么插入，找了很久都没有找到这个BN的matconvnet的deploy 例子，谢谢

### ISSUE THREAD

**Issue by yangkang779** [foolwood/DCFNet#20]

**Comment by yangkang779** [comment]
elseif isa(net.layers(i).block,'dagnn.ReLU')
                simple_net.layers{end+1} = struct(...
                    'name', deal(net.layers(i).name), ...
                    'type', 'relu') ;
            elseif isa(net.layers(i).block,'dagnn.LRN')
                simple_net.layers{end+1} = struct(...
                    'name', deal(net.layers(i).name), ...
                    'type', 'lrn',...
                    'param',net.layers(i).block.param) ;
            
            elseif isa(net.layers(i).block,'dagnn.BatchNorm')
                simple_net.layers{end+1} = struct(...
                    'name', deal(net.layers(i).name), ...
                    'type', 'bnorm') ;
but error

Reference to non-existent field 'filters'.

Error in vl_simplenn_tidy (line 62)
          layer.filters, ...

Error in run_DCFNet (line 7)
net = vl_simplenn_tidy(net.net);


Error in demo_DCFNet (line 15)
res = run_DCFNet(subS,0,0,param)
